src/platform/silabs/rs911x/BLEManagerImpl.cpp

```diff
@@ -93,10 +110,11 @@ void sl_ble_init()
     //  Exchange of GATT info with BLE stack
 
     rsi_ble_add_matter_service();
-
+    // Set the two least significant bits as the first 2 bits of the address has to be '11' to ensure the address is a random non-resolvable private address
+    randomAddrBLE[5] |= 0xC0;
```

Comment:
Move this after line number. 95.
So that it will not affect the other combos.

Reply:
Sure, made this change